src/test/kotlin/org/rust/ide/annotator/RsImplCopyRestrictionsTest.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.annotator
+
+class RsImplCopyRestrictionsTest : RsAnnotatorTestBase(RsErrorAnnotator::class) {
+    fun `test E0206 error when impl Copy for non-ADT`() = checkErrors("""
+        #[lang = "copy"]
+        trait Copy {}
+
+        struct Bar;
+
+        impl Copy for <error descr="The trait `Copy` may not be implemented for this type [E0206]">&'static mut Bar</error> { }
+    """)
+    fun `test E0206 error when impl Copy for trait`() = checkErrors("""
```

Comment:
Don't forget blank lines between tests
```suggestion
    """)

    fun `test E0206 error when impl Copy for trait`() = checkErrors("""
```